host/sgx/sgxload.c

```diff
@@ -385,14 +410,16 @@ oe_result_t oe_sgx_create_enclave(
     }
     else
     {
-        uint32_t enclave_error;
-        void* base = oe_sgx_enclave_create(
-            NULL, /* Let OS choose the enclave base address */
+        uint32_t enclave_error = 0;
+        base = oe_sgx_enclave_create_ex(
+            base_address,
```

Comment:
What's the definition of enclave_commit_size parameter for oe_sgx_enclave_create_ex()? Just the image size, or (image_size + base_address)?

Reply:
initial_commit_size is the physical memory to reserve for the initial loading of the enclave in bytes. All the common arguments between enclave_create() and enclave_create_ex() serve the same functions.

Reply:
Can you check if the `enclave_commit_size` can be only [start_address, elrange_end] instead of [elrang_start, elrange_end]? Ideally, we want the size represent the former range.

Reply:
@ming-wei done.